client/state/action-types.js

```diff
@@ -77,6 +77,10 @@ export const POST_EDITS_RESET = 'POST_EDITS_RESET';
 export const POST_REQUEST = 'POST_REQUEST';
 export const POST_REQUEST_SUCCESS = 'POST_REQUEST_SUCCESS';
 export const POST_REQUEST_FAILURE = 'POST_REQUEST_FAILURE';
+export const POST_STATS_RECEIVE = 'POST_STATS_RECEIVE';
+export const POST_STATS_REQUEST = 'POST_STATS_REQUEST';
+export const POST_STATS_REQUEST_SUCCESS = 'POST_STATS_REQUEST_SUCCESS';
+export const POST_STATS_REQUEST_FAILURE = 'POST_STATS_REQUEST_FAILURE';
```

Comment:
My fault for starting this trend, but `FAILURE` is alphabetically lesser than `SUCCESS`